### ISSUE TITLE
Curl import broken: "Got status 0 fetching schema from"

### ISSUE THREAD

**Issue by sgabler** [getinsomnia/insomnia#6405]
### Expected Behavior

When pasting a curl request into Insomnia, it should import the GraphQL request correctly.

### Actual Behavior

An error is shown and nothing is imported.

### Reproduction Steps

1. Open Insomnia
2. Click on the ➕ sign to add a new GraphQL Request
3. Copy any curl request from a GraphQL API into the address bar. I used this API in my example: https://studio.apollographql.com/public/countries/variant/current/explorer
4. A big error box appears with the following content:

![image](https://github.com/Kong/insomnia/assets/3625477/7d0ac715-b5e6-4e0e-98c4-276854189033)

```
Got status 0 fetching schema from "curl 'https://countries.trevorblades.com/graphql' --compressed -X POST -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/117.0' -H 'Accept: */*' -H 'Accept-Language: en-US,en;q=0.5' -H 'Accept-Encoding: gzip, deflate, br' -H 'Referer: https://studio.apollographql.com/' -H 'content-type: application/json' -H 'Origin: https://studio.apollographql.com' -H 'Connection: keep-alive' -H 'Sec-Fetch-Dest: empty' -H 'Sec-Fetch-Mode: cors' -H 'Sec-Fetch-Site: cross-site' -H 'TE: trailers' --data-raw '{"query":"query Query {\n country(code: \"BR\") {\n name\n native\n capital\n emoji\n currency\n languages {\n code\n name\n }\n }\n}\n","variables":{},"operationName":"Query"}'"
```

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

I tried 3 different APIs. All had the same problem

### Insomnia Version

2023.5.6

### What operating system are you using?

macOS

### Operating System Version

macOS Ventura 13.4 (22F66)

### Installation method

I think it was homebrew, but not 100% sure anymore

### Last Known Working Insomnia version

not sure, but I guess the previous one

**Comment by jackkav** [comment]
Should be fixed in #6384 Can you try again after we release 2023.5.7? After its imported change the body type from JSON to Graphql and it should work.

**Comment by sgabler** [comment]
@jackkav thanks, it works again with 2023.5.7 🎉

closing this issue.
